plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -310,6 +310,13 @@ export class DeploymentManager {
     return this.contractsCache;
   }
 
+  /* Gets all the contracts, connected to signer, as an object */
+  async getContracts(signer?: SignerWithAddress): Promise<{ [alias: Alias]: Contract }> {
```

Comment:
naming seems extremely similar to `this.contracts`

maybe `getConnectedContracts` or `getContractsWithSigner` to avoid confusion

Reply:
I actually think we should replace contracts with this but didn't want to start that yet, we can follow on later